src/terminal/interactive.rs

```diff
@@ -0,0 +1,26 @@
+// For interactively handling  reading in a string
+pub fn get_string(prompt_string: &str) -> String {
+    println!("{}", prompt_string);
+    let input: String = read!("{}\n");
+    input
+}
+
+// Truncate all "yes", "no" responses for interactive delete prompt to just "y" or "n".
+const INTERACTIVE_RESPONSE_LEN: usize = 1;
+const YES: &str = "y";
+const NO: &str = "n";
+// For interactively handling deletes (and discouraging accidental deletes).
+// Input like "yes", "Yes", "no", "No" will be accepted, thanks to the whitespace-stripping
+// and lowercasing logic below.
+pub fn delete(prompt_string: &str) -> Result<bool, failure::Error> {
+    println!("{} [y/n]", prompt_string);
+    let mut response: String = read!("{}\n");
+    response = response.split_whitespace().collect(); // remove whitespace
+    response.make_ascii_lowercase(); // ensure response is all lowercase
```

Comment:
is there a reason we use `make_ascii_lowercase` instead of `to_lowercase`?

Reply:
idk, @victoriabernard92 ?

Reply:
This was copy from gabbi's work on interactive delete

Reply:
interesting. it is oddly specific...perhaps we should leave it?

Reply:
i'm going to revert it to using this, and cc @gabbifish for when she gets back to see if we can change it. if not, i'd like to add a comment so we don't re-litigate this in the future.